google/resource_container_cluster.go

```diff
@@ -1823,3 +1826,21 @@ func masterAuthClientCertCfgSuppress(k, old, new string, r *schema.ResourceData)
 
 	return strings.HasSuffix(k, ".issue_client_certificate") && old == "" && new == "true"
 }
+
+// We want to suppress diffs for empty/disabled private cluster config.
+func containerClusterPrivateClusterConfigSuppress(k, old, new string, d *schema.ResourceData) bool {
+	o, n := d.GetChange("private_cluster_config.0.enable_private_endpoint")
```

Comment:
So just to clarify, this will return `false` if there's no `private_cluster_config` block?

Reply:
yup!